.github/workflows/copy-workflow.yml

```diff
@@ -12,43 +12,62 @@ jobs:
       matrix:
         cfg: ${{ github.event.client_payload.targets }}
     env:
+      TARGET_REPO_DIR: "target-repo"
       TEMPLATE_REPO_DIR: "template-repo"
       TEMPLATE_DIR: "templates"
       NEEDS_UPDATE: 0
       INITIAL_TEST_DEPLOYMENT: 0
+      GITHUB_USER: ""
+      GITHUB_EMAIL: ""
```

Comment:
I imagine this still works without these zero initial values? The same applies to NEEDS_UPDATE and others.

Reply:
Unfortunately it doesn't, I tried it. GitHub Actions seems to keep environment variables tightly scoped: If you don't declare it for the job, it will only be available in the step where you set it.